VMs/metrics/ubuntu16.04/user-data

```diff
@@ -0,0 +1,54 @@
+#cloud-config
+# Copyright (c) 2018 Intel Corporation
+# 
+# SPDX-License-Identifier: Apache-2.0
+
+groups:
+  - jenkins
+
+users:
+  - name: USERNAME
+    gecos: Jenkins User
+    lock-passwd: false
+    shell: /bin/bash
+    sudo: ALL=(ALL) NOPASSWD:ALL
+    ssh-authorized-keys:
+      - PUBLICKEYCONTENTS
+
+packages:
+ - gawk
+ - bc
+ - curl
+ - git
+ - jq
+ - default-jre
+ - gcc
+ - make
+ - build-essential
+
+runcmd:
+
+ - rm /etc/update-motd.d/10-help-text /etc/update-motd.d/51-cloudguest
+ - rm /etc/update-motd.d/90-updates-available
+ - rm /etc/legal
+
+# Install Go
+# The sourceing of environment is a workaround to get any proxy settings we need
+ - 'bash -c "set -a; source /etc/environment; set +a; curl -L https://dl.google.com/go/go1.10.2.linux-amd64.tar.gz -o /tmp/go1.10.2.linux-amd64.tar.gz"'
+ - tar -C /usr/local -xzf /tmp/go1.10.2.linux-amd64.tar.gz
+ - rm /tmp/go1.10.2.linux-amd64.tar.gz
+
+# Install docker
+# The sourceing of environment is a workaround to get any proxy settings we need
+ - 'bash -c "set -a; source /etc/environment; set +a; curl -fsSL https://download.docker.com/linux/ubuntu/gpg | sudo apt-key add -"'
+ - sudo add-apt-repository "deb [arch=amd64] https://download.docker.com/linux/ubuntu $(lsb_release -cs) stable"
```

Comment:
Hard-coded arch.

Reply:
I think I'm going to say 'we are very happy for PRs to update this'... I'll have a look to see if there is a trivial fix, otherwise I think it will stay for the minute

Reply:
ok. Now we have three architectures, we really need to try to make all new code architecture agnostic or tolerant atleast. However, if there is urgency to land this or we just aren't sure how best to resolve it for the other arches, how about raising an issue so we don't forget to update this (you could also /cc some folks maybe so they are aware in advance that this is limited to a single arch at this stage).

Reply:
I've added #35, and also added a check so we die at the moment if we are not on x86.